docs/hmm.rst

```diff
@@ -22,16 +22,16 @@ or `mixture models <http://en.wikipedia.org/wiki/Mixture_model>`_, which
 smooth the data in a potentially more natural way.
 
 A Gaussian hidden Markov model (HMM) is one way of applying this same logic
-to probabilistic models of the dynamics of molecular system. Like
+to probabilistic models of dynamics of molecular systems. Like
 :ref:`MSMs <msm>`, the HMM also models the dynamics of the system as a
-first-order Markov jump process between discrete set of states. The
+first-order Markov jump process between a discrete set of states. The
 difference is that the states in the HMM are not associated with discrete
-non-overlapping regions of phase space defined by clustering -- instead the
+non-overlapping regions of phase space defined by clustering -- instead, the
 states are Gaussian distributions. Because the Gaussian distribution has
-infinite support, there is no unique and unambiguous mapping from
-conformation to state. Each state is a distribution over all conformations.
+infinite support, there is no unique and unambiguous state mapping to
+a given conformation. Each state is a distribution over all conformations.
```

Comment:
This changes the logic, no? It used to be conformation --> state, and now it's state --> conformation